parameter/ParameterMgr.cpp

```diff
@@ -93,15 +90,27 @@
 #include <sstream>
 #include <algorithm>
 #include <ctype.h>
-#include <memory>
 
 #define base CElement
 
+/** Private macro helper to declare a new context
+ *
+ * Context declaration always need logger and logging prefix to be
+ * passed as parameters.
+ * This macro aims to avoid this boring notation.
+ * This macro should be called only once in a scope.
```

Comment:
This is not completely accurate. Consider the following code:

```
{
  // Scope a
  LOG_CONTEXT(...);

  {
    // Scope b
    LOG_CONTEXT(..);
  } // End of scope b
} // End of scope a
```

This code is fine even though the macro is called twice in Scope `a`. However, I'm not sure how to phrase this... Maybe "This macro may only be called once in a scope's root but may be called again in nested scopes." ?